### PR TITLE
[RNMobile] Fix unintended home button trigger on iOS in e2e tests

### PR DESCRIPTION
**Pull request by ceyhun**
## Description

Moved drag y point to scroll block list higher to avoid triggering home button.

## How has this been tested?

Locally: `npm run native test:e2e:ios:local gutenberg-editor-latest-posts`
CI: Check if all e2e tests pass on gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2926

## Screenshots 

Before | After
-- | -- 
|

## Types of changes

Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/react-native-editor/__device-tests__/pages/editor-page.js [hunk 1]
```diff
@@ -245,7 +245,7 @@ class EditorPage {
 			blockAccessibilityLabel
 		);
 		const size = await this.driver.getWindowSize();
-		const height = size.height - 5;
+		const height = size.height - 50;
```
Comment (hypest) [inline packages/react-native-editor/__device-tests__/pages/editor-page.js:248]: For future reference, what's the typical safe area height we assume at this point, and what's the block picker height? Trying to understand if 50 is still very close to the safe area or not.

Comment (hypest) [reply packages/react-native-editor/__device-tests__/pages/editor-page.js:248]: While at it, I think it makes sense to add a line comment about the hardcoded value, explaining a bit how we decided on the specific value and what's its role.

Comment (ceyhun) [reply packages/react-native-editor/__device-tests__/pages/editor-page.js:248]: Safe are is 34 in portrait and 21 on landscape. The picker is 464 in iPhone 11 (the one used in e2e tests).

**Review by hypest — APPROVED** [review]
Only left 2 minor comments (one of which you've already addressed Ceyhun) so, this LGTM! Feel free to merge after adding a line comment for the hardcoded value 👍, thanks!

**Pull request merged**